Account key link transaction the remotePublicKey property was rename to linkedPublicKey

For account key link transaction, the remotePublicKey property need to get rename to linkedPublicKey.  This is causing failures in the SDK.

https://github.com/nemtech/symbol-openapi/blob/51bd06363a972489f3df34ea168fd4df4c54c4c7/spec/plugins/account_link/schemas/AccountKeyLinkTransactionBodyDTO.yml#L6

{"meta":{"height":"333","hash":"86D317C374E4552A27850A3CC5776446903835CFCD75740FBBDBB483AA5EE05C","merkleComponentHash":"86D317C374E4552A27850A3CC5776446903835CFCD75740FBBDBB483AA5EE05C","index":0,"id":"5EC4505DB8551373BD6D227B"},"transaction":{"signature":"837A63D8869F7D3C28983B273A580BC46CD1B4A395954A9DC96FE2F99527CEF756DF463CF695D7B3C1426834F11C69FF8180D8FEF37211577EB8AC124A229004","signerPublicKey":"89589C81C500A65F4019DAD36198230373EBC2C646D3FDD0DDD3D966404559D4","version":1,"network":152,"type":16716,"maxFee":"16100","deadline":"16500718888","linkedPublicKey":"cbJOSD9mbSB8tSpqbYyAw/60wWIFVGji0vtY1LDuzWc=","linkAction":1}}


Caused by: java.lang.IllegalArgumentException: Public key is not valid
	at io.nem.symbol.core.crypto.RawAddress.generateAddress(RawAddress.java:56) ~[symbol-sdk-core-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.model.account.Address.createFromPublicKey(Address.java:144) ~[symbol-sdk-core-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.model.account.PublicAccount.<init>(PublicAccount.java:34) ~[symbol-sdk-core-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.model.account.PublicAccount.createFromPublicKey(PublicAccount.java:46) ~[symbol-sdk-core-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.infrastructure.vertx.mappers.AccountKeyLinkTransactionMapper.createFactory(AccountLinkTransactionMapper.java:43) ~[symbol-sdk-vertx-client-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.infrastructure.vertx.mappers.AccountKeyLinkTransactionMapper.createFactory(AccountLinkTransactionMapper.java:32) ~[symbol-sdk-vertx-client-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.infrastructure.vertx.mappers.AbstractTransactionMapper.createModel(AbstractTransactionMapper.java:75) ~[symbol-sdk-vertx-client-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.infrastructure.vertx.mappers.AbstractTransactionMapper.map(AbstractTransactionMapper.java:67) ~[symbol-sdk-vertx-client-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.infrastructure.vertx.mappers.GeneralTransactionMapper.map(GeneralTransactionMapper.java:89) ~[symbol-sdk-vertx-client-0.17.2-20200516.002415-7.jar:?]
	at io.nem.symbol.sdk.infrastructure.vertx.TransactionRepositoryVertxImpl.toTransaction(TransactionRepositoryVertxImpl.java:76) ~[symbol-sdk-vertx-client-0.17.2-20200516.002415-7.jar:?]
	at io.reactivex.internal.operators.observable.ObservableMap$MapObserver.onNext(ObservableMap.java:59) ~[rxjava-2.1.7.jar:?]
	at io.reactivex.internal.operators.observable.ObservableOnErrorNext$OnErrorNextObserver.onNext(ObservableOnErrorNext.java:68) ~[rxjava-2.1.7.jar:?]
	at io.reactivex.internal.operators.single.SingleToObservable$SingleToObservableObserver.onSuccess(SingleToObservable.java:59) ~[rxjava-2.1.7.jar:?]
	at io.vertx.reactivex.core.impl.AsyncResultSingle.lambda$subscribeActual$0(AsyncResultSingle.java:41) ~[vertx-rx-java2-3.5.0.jar:?]
	at io.nem.symbol.sdk.openapi.vertx.invoker.ApiClient.lambda$buildResponseHandler$7(ApiClient.java:599) ~[symbol-openapi-vertx-client-0.8.10.jar:?]
	at io.vertx.ext.web.client.impl.HttpContext.lambda$null$0(HttpContext.java:125) ~[vertx-web-client-3.5.0.jar:?]

--- a/spec/plugins/account_link/schemas/AccountKeyLinkTransactionBodyDTO.yml
+++ b/spec/plugins/account_link/schemas/AccountKeyLinkTransactionBodyDTO.yml
@@ -3,7 +3,7 @@ required:
   - linkedPublicKey
   - linkAction
 properties:
-  remotePublicKey:
+  linkedPublicKey:
     $ref: "../../../schemas/PublicKey.yml"
   linkAction:
     $ref: "../../../schemas/LinkActionEnum.yml"

--- a/spec/plugins/account_link/schemas/AccountKeyLinkTransactionBodyDTO.yml
+++ b/spec/plugins/account_link/schemas/AccountKeyLinkTransactionBodyDTO.yml
@@ -1,6 +1,6 @@
 type: object
 required:
-  - remotePublicKey
+  - linkedPublicKey
   - linkAction
 properties:
   remotePublicKey:

